common/uri/src/test/java/io/helidon/common/uri/UriQueryTest.java

```diff
@@ -54,4 +54,13 @@ void testEncodedOtherChars() {
         assertThat(uriQuery.value("h"), is("xc#e<"));
         assertThat(uriQuery.value("a"), is("b&c=d"));
     }
+    
+    @Test
+    void testEmptyQueryString() {
+        UriQuery uriQuery = UriQuery.create(null);
```

Comment:
`UriQuery.create(null)` should not be supported and should throw a NPE.

Reply:
If you feel this strongly that null should not be allowed as the value for `UriQuery` then let's approve this very focused bug fix as-is and create a separate issue for null handling.

There are several places in `UriQueryImpl` (unchanged by this PR) that check for null and take relevant action, so the original author (!) seems to have considered this a valid use case and there could be code elsewhere that depends on this behavior. 

I would rather not expand the scope of _this_ bug fix to address other aspects of `UriQuery` that seem to be working as designed.

Reply:
The original author understands.